vuepress/docs/next/tutorials/devops/plugin-configuration.md

```diff
@@ -7,11 +7,8 @@ sidebarDepth: 2
 This tutorial describes three methods to utilize configuration profiles to specify resource allocation for Entando microservices. These provide a simple way to customize microservice deployment parameters for improved efficiency.
 
 ## Prerequisites
-* [Add an Entando Operator ConfigMap](../consume/entando-operator.md) if needed, then
-* Enable this property under the `data` section so that the Entando Operator can manage resource settings:
-```yaml
- entando.k8s.operator.impose.limits: "true"
```

Comment:
Why was this removed? Was this a change in 7.2?

Reply:
@jyunmitch Maybe I'm missing something but I don't think it's there by default. Definitely not in the template we provide https://github.com/entando/entando-releases/blob/release/7.2/dist/ge-1-1-6/samples/entando-operator-config.yaml or in the custom resources. I think it's automatically applied by AWC when we install into PS Dev but that isn't the default config. What do you think?

Reply:
@nshaw, you're completely right. I assumed that our Rancher test set up would be the default one. If that's not the default one, how can I know what the customers see

Reply:
@jyunmitch Great question. It might be worth asking PJ or Sergio if there's a document on how the PS Dev env differs from a stock install. That might already exist in notion. The alternative is to check a vanilla install on multipass or check the code, like I just did, but that's a pain.